src/Grid.js

```diff
@@ -1,24 +1,33 @@
 import Cell from './Cell';
 
 export default class Grid {
-    constructor() {
-        this.height = 50;
-        this.width = 50;
-        this.grid = this.generateGrid();
+    constructor(width, height) {
+        this.height = height;
```

Comment:
<sub>turbo nit: we should match arg order</sub>

Reply:
🐇 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡 😡